File: app/src/main/java/com/deniscerri/ytdlnis/ui/MoreFragment.java
package com.deniscerri.ytdlnis.ui;

import android.os.Bundle;
import androidx.preference.PreferenceFragmentCompat;
import com.deniscerri.ytdlnis.R;

public class MoreFragment extends PreferenceFragmentCompat {
    public static final String TAG = "MoreFragment";
    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.more_preferences, rootKey);
    }
}

File: app/src/main/java/com/deniscerri/ytdlnis/ui/MoreFragment.kt
package com.deniscerri.ytdlnis.ui

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.deniscerri.ytdlnis.R

class MoreFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.more_preferences, rootKey)
    }

    companion object {
        const val TAG = "MoreFragment"
    }
}